shared/Cargo.toml

```diff
@@ -9,7 +9,15 @@ version = "0.1.0"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
-default = []
+default = [
+  "tpke",
+  "ark-bls12-381",
+  "ark-ec",
+  "ark-serialize",
+  "rand_core",
+  "rand_new"]
+
+wasm = []
```

Comment:
this is not used, is it?

Reply:
Oops, that is a relic of an abandoned idea